btx/diagnostics/run.py

```diff
@@ -26,7 +26,8 @@ def __init__(self, exp, run, det_type):
         self.stats = dict()
         self.gain_traj = None
         self.gain_map = None
-        
+        self.gain_mode = ''
```

Comment:
Was this change made just for compatibility with f-strings? It's not immediately clear to me why it was necessary - not really important, just curious :)

Reply:
I chose to initialize it here because I got an error thrown by https://github.com/lcls-users/btx/blob/44c7291d9d5d1d78cbc5fd7b52e7348b79785024/btx/diagnostics/run.py#L328

Maybe there is a better solution?

Reply:
Makes sense. I think this is probably the easiest way.